simple/stats/runner.py

```diff
@@ -44,6 +46,8 @@ def __init__(
     self.output_dir_fh = create_file_handler(output_dir)
     self.process_dir_fh = self.output_dir_fh.make_file(
         f"{constants.PROCESS_DIR_NAME}/")
+    self.reporter = ImportReporter(report_fh=self.process_dir_fh.make_file(
+        constants.REPORT_JSON_FILE_NAME))
```

Comment:
Shall we name the report file with timestamp is it's not overwritten?

Reply:
I'm thinking since the mixer load endpoint will call the script with a new output dir on every execution, we won't run into the overwrite problem?